trust-gh-pages/js/sims/PD.js

```diff
@@ -333,9 +378,15 @@ function Logic_prober(){
 // ORIGINAL RH LOGIC (not hooked up to any current player)
 function Logic_robinhood(){
   var self = this;
+  var AI = false;
```

Comment:
Any potential upsides to storing it here rather than in the TournamentAgent object? Just curious